PatientDiagnosis/patients/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+from examinations.models import Examination
+
+
+class Patient(models.Model):
+    surname = models.CharField(max_length=100)
+    age = models.PositiveIntegerField(default=0)
+    examination = models.ForeignKey(Examination, on_delete=models.DO_NOTHING, related_name='patients')
```

Comment:
on_delete=models.DO_NOTHING jest przemyślane? Zazwyczaj robi się CASCADE (tu też chyba to ma sens, bo jeśli usuwamy pacjenta to badania chyba też) ale jeśli zrobiłeś to umyślnie to w porządku

Reply:
Zrobiłem umyślnie. Nie wiem jak to będzie docelowo, ale wyobrażam sobie, że dla celów statystycznych mimo wszystko warto zostawić badania.